### ISSUE TITLE
Dart/Node SASS has deprecated / as division operator

### ISSUE THREAD

**Issue by mikniin** [foundation/foundation-sites#12237]
Hi,

As the title says, the '/' is in dart-sass 2.0 onwards only a separator. Right now it has been deprecated and will be removed in version 2.

Currently getting loads of spam like this when compiling CSS with Foundation sites 6.6.3:

```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(rem-calc($gutters), 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
38 │     $gutter: rem-calc($gutters) / 2;
   │              ^^^^^^^^^^^^^^^^^^^^^^
   ╵
    node_modules/foundation-sites/scss/xy-grid/_gutters.scss 38:14  xy-gutters()
    node_modules/foundation-sites/scss/xy-grid/_grid.scss 17:3      xy-grid-container()
    node_modules/foundation-sites/scss/xy-grid/_classes.scss 21:7   xy-base-grid-classes()
    node_modules/foundation-sites/scss/xy-grid/_classes.scss 457:5  foundation-xy-grid-classes()
    scss/bot.scss 10:1                                              root stylesheet
```

**Comment by toazter** [comment]
Yeah I've had this happen for a few days now. The docs at `https://sass-lang.com/d/slash-div` suggest running their migrator to fix the issue across all SCSS files. Any chance there will be a minor version to resolve this? 

It's not breaking anything yet. However, it is rather annoying scrolling through 15+ of these errors to get to something you needed to log through node.

**Comment by rdavid2** [comment]
Totally agree, foundation version 7 is coming, but version 6 is still in many active projects. 

+1 to a smaller version that fixes these issuess

**Comment by ahukkanen** [comment]
Just a note to add to this if the team wants to preserve compatibility with LibSass for version 6.

There is a related discussion at Bootstrap repo:
https://github.com/twbs/bootstrap/issues/34353

There it was mentioned that Bootstrap provides a `divide` function to ease the transition:
https://github.com/twbs/bootstrap/blob/5c1691ac3a7511e3ae0f1f619bfc0c848090efb5/scss/_functions.scss#L233-L273

It could be borrowed to Foundation as well (licenses match). There are many places in Foundation which could be just replaced with multiplications (e.g. `num / 2` = `num * 0.5`) which the `sass-migrator` tool correctly does but for the remaining divisions, the `divide` function could be borrowed from Bootstrap.

**Comment by ahukkanen** [comment]
I opened a PR for fixing this issue: #12241

**Comment by xavierfoucrier** [comment]
Hi everyone,

This is related to https://github.com/foundation/foundation-sites/issues/12233.
It would be good to have a new minor version published to fix that.

@joeworkman, can you take a look at the @ahukkanen pull request?

Thanks! :wink:

**Comment by newhouse** [comment]
bump?

**Comment by nolafs** [comment]
Guys, i am running build pipe which is actually failing because of those warnings. Please approve the pull request

**Comment by joeworkman** [comment]
Ok everyone, can you please test out the `develop` branch. This is merged in along with a bunch more things that I have been delaying. I am sure that it's not a big deal to most of you here, but node 12 is required now. I was not able to get things running smoothly under node 16 however. So v12 or v14 worked fine in my tests.

**Comment by mikaeldiener** [comment]
@joeworkman Will the new release v6.7.0, be released as a npm package?

**Comment by joeworkman** [comment]
Yes. I ran out of time yesterday. I just want to triple check a few things on Monday before doing so.
